pyproject.toml

```diff
@@ -4,9 +4,7 @@ version = "0.2.7"
 description = "The official Python library for the llama-stack-client API"
 dynamic = ["readme"]
 license = "Apache-2.0"
-authors = [
-{ name = "Llama Stack Client", email = "dev-feedback@llama-stack-client.com" },
-]
+authors = [{ name = "Meta Llama", email = "llama-oss@meta.com" }]
```

Comment:
this is cleanup for pyproject which makes it workable in the IDE otherwise it is filled with all redlines because pyright sucks